features/docs/dan-stap-definities-problem-details.feature

```diff
@@ -51,7 +51,7 @@ Functionaliteit: Problem Details dan stap definities
       ]
     }
     """
-    Dan heeft de response invalidParams met de volgende gegevens
+    Dan de volgende invalidParams foutmeldingen
```

Comment:
voorstel: "Dan zijn er de volgende niet valide parameters"

of stappen we dan weer té ver van de implementatie/interface af?

Reply:
aangepast naar "Dan parameter foutmeldingen"